src/Adapters/MockAdapter.ts

```diff
@@ -681,16 +681,47 @@ export default class MockAdapter
         }
     }
 
-    async advancedSearchADTTwins(
-        _params: AdapterMethodParamsForAdvancedSearchADTwins
-    ) {
+    getFirstPropertyFromQuery = (query: string) => {
+        // Initial position index is the index after WHERE in the query
+        // used here to search for first property after the WHERE clause
+        const initialPositionIndex = 43;
```

Comment:
i know it's only for mock data, but can we at least make it resilient to a single character change? Can we just look up the position of `where`?